administrator/language/de-DE/plg_schemaorg_article.ini

```
@@ -0,0 +1,29 @@
+; Joomla! German Translation
+; (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2024 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_ARTICLE="Schema.org - Artikel"
+PLG_SCHEMAORG_ARTICLE_DESCRIPTION_LABEL="Weitere Informationen über den Schema.org-Artikeltyp sind unter <a href='https://schema.org/Article' target='_blank' rel='noopener noreferrer'>https://schema.org/Article</a> zu finden."
+PLG_SCHEMAORG_ARTICLE_FIELD_ADDRESS_LABEL="Adresse"
+PLG_SCHEMAORG_ARTICLE_FIELD_AUTHOR_LABEL="Autor"
+PLG_SCHEMAORG_ARTICLE_FIELD_DATE_MODIFIED_LABEL="Änderungsdatum"
+PLG_SCHEMAORG_ARTICLE_FIELD_DATE_PUBLISHED_LABEL="Veröffentlichungsdatum"
+PLG_SCHEMAORG_ARTICLE_FIELD_DESCRIPTION_LABEL="Beschreibung"
+PLG_SCHEMAORG_ARTICLE_FIELD_EMAIL_LABEL="E-Mail"
+PLG_SCHEMAORG_ARTICLE_FIELD_GENERIC_FIELD_LABEL="Allgemeines Feld"
+PLG_SCHEMAORG_ARTICLE_FIELD_GENERIC_TITLE_LABEL="Titel"
```

```suggestion
PLG_SCHEMAORG_ARTICLE_FIELD_GENERIC_TITLE_LABEL="Bezeichnung"
```
vgl. andere Schema.org-Plugins